src/Provider/Provider.Web/Views/CloneVacancy/CloneVacancyWithNewDates.cshtml

```diff
@@ -0,0 +1,73 @@
+@model Esfa.Recruit.Provider.Web.ViewModels.CloneVacancy.CloneVacancyWithNewDatesViewModel
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState, OrderedFieldNames = Model.OrderedFieldNames }' />
+        <h1 class="govuk-heading-xl">@Model.Title</h1>
+        <div asp-show="Model.IsNewDatesForced" class="govuk-inset-text">The vacancy you are copying has a closing date or start date that is in the past. Add new dates.</div>
+        <form asp-route="@RouteNames.CloneVacancyWithNewDates_Post" novalidate>
+            <div esfa-validation-marker-for="ClosingDate" class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
+                    <span esfa-validation-message-for="ClosingDate" class="govuk-error-message"></span>
+                    <legend class="govuk-fieldset__legend govuk-!-font-weight-bold">Application closing date</legend>
+                    <span id="closing-hint" class="govuk-hint">
+                        For example, 19 01 2019
```

Comment:
If you see our Part 1 Training view, you will see we are adding a non static example date. Could you do the same here, prevents us needing to update copy

Reply:
fixed